vault/namespaces.go

```diff
@@ -17,3 +17,9 @@ func namespaceByID(ctx context.Context, nsID string, c *Core) (*namespace.Namesp
 	}
 	return nil, namespace.ErrNoNamespace
 }
+
+var NamespaceRegister func(context.Context, *namespace.Namespace, *Core) error = namespaceRegister
```

Comment:
not a blocker, but you can instead add a function to a [stubs_oss.go file](https://github.com/hashicorp/vault/blob/main/vault/core_stubs_oss.go) if you want to use the stubmaker to handle generating an enterprise function before the enterprise PR is merged. But this isn't required, and your method works as well.

Reply:
Thanks Mia. Yeah this is not the newest/best pattern we use for this but I thought I'd follow the pattern of `NamespaceByID` above for now as they are closely related and live in the same file.